pkg/security/common/containerutils/utils.go

```diff
@@ -24,7 +24,7 @@ var wildContainerIDPattern = regexp.MustCompilePOSIX(WildContainerIDPatternStr)
 
 // FindContainerID extracts the first sub string that matches the pattern of a container ID
 func FindContainerID(s string) string {
-	if strings.Contains(s, "docker") {
+	if strings.Contains(s, "/docker/") || strings.Contains(s, "/kubepods.slice/") {
```

Comment:
What would happen for cri-o, or podman or any other container system ?

Reply:
Or even "bare" containerd not as a container engine for k8s.